rpc-client-interface.go

```diff
@@ -29,6 +29,9 @@ type LockArgs struct {
 
 	// ServiceEndpoint contains the network path of above server to do lock/unlock.
 	ServiceEndpoint string
+
+	// Source contains the source on the client node that requested the lock.
```

Comment:
Is the source a source file, function name? we should perhaps make it a bit more clear..

Reply:
done